web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -744,6 +749,13 @@ class SpeakPage extends React.Component<Props, State> {
                 label: 'shortcut-rerecord-toggle-label',
                 action: this.handleRecordClick,
               },
+              {
+                key: 'shortcut-discard-ongoing-recording',
+                label: 'shortcut-discard-ongoing-recording-label',
+                icon: <span>⌫</span>,
```

Comment:
A proper icon would be nice, but I'm no artist :) If we can't trust that all fonts have this unicode symbol, a textual representation might be better (but "BACKSPACE" overflows the shortcutbox, so it's will require some work)

Reply:
If we're changing to ESC, I think just using `ESC` would be fine for the "icon"

Reply:
Agreed